### DISCUSSION TITLE
Show existing item url

### DISCUSSION THREAD

**Discussion by soredake** [bibanon/tubeup#319] [Ideas]
It would be nice if tubeup can show existing item url.

Right now, it does not showing it:
![image](https://github.com/bibanon/tubeup/assets/5204968/060469d4-e390-4136-9640-69e0e360d91d)

**Comment by mrpapersonic** [comment]
This was discussed in #264 but was never put into fruition. It seems easy enough to add, I'll look into it when I have the time.

**Comment by vxbinaca** [comment]
> This was discussed in #264 but was never put into fruition. It seems easy enough to add, I'll look into it when I have the time.

Thank you for the quick reply. This too bugged me slightly. However, why not both? Both the source video URL and the item.
